main_taskqueue.py

```diff
@@ -24,8 +24,11 @@
 
 import webapp2
 
+MYPY = False
```

Comment:
Can we have this as 1 flag instead of multiple? Provides a single point of control, and easily enabling/disabling this flag.

Reply:
Actually this is not a flag. This must be false in our codebase. Reason:
The `MYPY` flag should be false throughout our code. And while type checking, mypy assumes it to be True. If we want to set it to true, it will be better to just remove this variable. Thanks!